fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt

```diff
@@ -232,10 +232,16 @@ class HttpClient(
         val contentLength = body.length
         if (contentLength != null && contentLength != -1L) {
             // The content has a known length, so no need to chunk
-            connection.setFixedLengthStreamingMode(contentLength.toLong())
```

Comment:
This is supposed to set the content length as well.

Reply:
it never set the Header for it.

Reply:
If you're certain (check in the server; mock api, not in the headers object before it's sent), then this is fine with me.